DashWallet/Sources/Models/CrowdNode/CrowdNode.swift

```diff
@@ -758,11 +800,40 @@ extension CrowdNode {
                 notifyIfNeeded(message: NSLocalizedString("Your CrowdNode address has been validated, but verification is required.", comment: "CrowdNode"))
             } else if status.lowercased() == kConfirmedStatus {
                 changeOnlineState(to: .done)
-                notifyIfNeeded(message: NSLocalizedString("Your CrowdNode address has been confirmed.", comment: "CrowdNode"))
+                
+                if prefs.shouldShowConfirmedNotification {
```

Comment:
Actually, I forgot about this case, so there was probably a bug with the previous solution.